.idea/housing-search-listener.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is this used for?

Reply:
File created from IDE.  File has been removed.